### DISCUSSION TITLE
Is it possible to render only one .qmd file in a blog post?

### DISCUSSION THREAD

**Discussion by ManuelSpinola** [quarto-dev/quarto-cli#3520] [Q&A]
I am working on a blog made with quarto.

Is it posiible to render only the .qmd file that contain the content of the post and avoid rendering all the .qmd files in other folders of the post folder?

In the post folder I keep other files that contain other files including some .qmd files to do some content for the post.

**Comment by dragonstyle** [comment]
There should be a way to accomplish this (depending upon exactly what the structure of the files is). Here is more on controlling what is rendered in a project:

https://quarto.org/docs/projects/quarto-projects.html#render-targets

**Reply by ManuelSpinola** [reply]
Thank you very much Charles.  That answer my question.

**Reply by ManuelSpinola** [reply]
However, following your advive even when I specify wich .qmd I want to render stiil is rendering unwanted .qmd files.

**Reply by dragonstyle** [reply]
Sorry to hear this! Can you be a little more specific about how you have things set up and what specifically isn't working?

**Comment by mcanouil** [comment]
The freeze option will also skip up to date documents.
See <https://quarto.org/docs/projects/code-execution.html#freeze>.

**Reply by ManuelSpinola** [reply]
I did several things, including deleting the post from the _freeze folder but I still cannot render the file.

I asked specifically not to read the file plot_3d.qmd stored in the folder plot_3d

project:
  render:
    - lulc_cr_2021.qmd
    - "!plot_3d/"

Preparing to preview
[1/4] posts/2022-11-24-lulc-cr-2021/plot_3d/plot_3d.qmd

Quitting from lines 65-66 (plot_3d.qmd) 
Error: [writeRaster] cannot write file

Execution halted

**Reply by dragonstyle** [reply]
I'm not sure why that is. Here is a sample project that worked for me, skipping rendering of a `qmd` in a subfolder. HTH!

[blog.zip](https://github.com/quarto-dev/quarto-cli/files/10125898/blog.zip)

**Reply by ManuelSpinola** [reply]
Thank you very much.

I see, but this is in the yaml of the entire blog (_quarto.yml).  I was thinking that you can specify to avoid rendering some files in the specific post yaml.

**Reply by dragonstyle** [reply]
I'd go with the approach of prefixing the file names with e.g. `_plot_3d.qmd`  - I think that will work best for your scenario.

**Reply by ManuelSpinola** [reply]
Thank you very much.  Using the arguments in the _quarto.yml of the blog works.  I was using the arguments in the YAML in the .qmd file of the post.
